### PR TITLE
carousel

### PR DESCRIPTION
**Pull request by arshakpoghosyan**
adding movie slideshow

### CODE REVIEW

**Review by artaksafaryan — APPROVED** [review]

File: css/carousel.css [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+        display: none;
+      } 
+      .small-about-film2{
+        display: block;
+      }
+      .film-header{
+        font-size: 20px;
+      }
+  }
+
```
Comment (mikayel00) [inline css/carousel.css:272]: add new line

File: home.html [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                  <div class="film-header">Spider-Man: No Way Home (2021)</div>
+                  <a class="film-genre">Action,Adventure,Fantasy • 2h 28m</a>
+                  <p class="about-film">With Spider-Man's identity now revealed, Peter asks Doctor Strange for help. When a spell goes wrong, dangerous foes from other worlds start to appear, forcing Peter to discover what it truly means to be Spider-Man.</p>
+                  <p class="small-about-film">With Spider-Man's identity now revealed, Peter asks Doctor Strange for help. When a spell goes wrong, dangerous...</p>
+                  <p class="small-about-film2">With Spider-Man's identity now revealed, Peter asks Doctor Strange for help. When a spell...</p>
+                  <button onclick="window.location.href='ticket.html'" class="get-ticket">Get Tickets</button>
+                  <button class="ticket-price">2500 AMD/3500 AMD</button>
+                  <button class="more-about">More about movie</button>
+                </div>
+
```
Comment (lilithkarapetyan) [inline home.html:127]: Can we make this mySlides part dynamic to avoid code duplication ?

File: css/carousel.css [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+  .slideshow-container {
+    max-width: 1000px;
+    position: relative;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    z-index: 3;
+  }
+  
+  .prev, .next {
```
Comment (robhov97) [inline css/carousel.css]: you can combine this in a one common class

File: home.html [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    <link rel="stylesheet" href="css/global.css">
+    <link rel="stylesheet" href="css/header.css">
+    <link rel="stylesheet" href="css/footer.css">
+    <link rel="stylesheet" href="css/carousel.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;700&family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+    <title>Buy Ticket | Coliseum Cinema</title>
+</head>
+
+<body>
+    <div class="wrapper">
```
Comment (robhov97) [inline home.html:17]: if this div just wraps all the content inside of the body, you can just remove it and apply style for wrapper to the body

File: js/carousel.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+showSlides(slideIndex);
+
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  let slides = document.getElementsByClassName("mySlides");
+  if (n > slides.length) {slideIndex = 1}    
+  if (n < 1) {slideIndex = slides.length}
```
Comment (robhov97) [inline js/carousel.js:11]: You can avoid brackets if you are writing the if's block in one line, but you can leave it as it is, that's okay too.
```suggestion
  if (n > slides.length) slideIndex = 1    
  if (n < 1) slideIndex = slides.length
```

**Review by robhov97 — APPROVED** [review]

**Pull request merged**